src/chapters/ldap-auth.xml

```diff
@@ -356,6 +356,32 @@ dn: cn={4}guacConfigGroup,cn=schema,cn=config
                             "<systemitem>uid=user,ou=people,dc=example,dc=net</systemitem>".</para>
                     </listitem>
                 </varlistentry>
+                <varlistentry>
+                    <term><property>ldap-user-attributes</property></term>
+                    <listitem>
+                        <para>The attribute or attributes to retrieve from the LDAP directory
+                            for the currently logged-in user. These attributes are stored as
+                            tokens with the prefix "GUAC_ATTR_" and the name of the attribute appended
+                            in uppercase letters. The value of the token is the value of the attribute
+                            in the LDAP directory for the currently logged-in user. If the attribute
+                            has no value in the LDAP directory then the token is not saved.
+                            If the attribute has multiple values in the LDAP directory then the token
+                            saves the first value of the attribute. Multiple attributes can be
+                            specified here, separated by commas.</para>
+                        <para>For example, if <property>ldap-user-attributes</property> is
+                        "<systemitem>mail, workstation</systemitem>", then a GUAC_ATTR_MAIL
```

Comment:
Formatting here should be consistent with the paragraph above- that is the `<para>` tag is good where it is, but the next lines should be indented.